src/Notifier.js

```diff
@@ -203,12 +205,10 @@ var Notifier = {
     },
 
     onSyncStateChange: function(state) {
+        console.log("sync state change: " + state);
         if (state === "PREPARED" || state === "SYNCING") {
             this.isPrepared = true;
         }
-        else if (state === "STOPPED" || state === "ERROR") {
```

Comment:
I think this will also change the behaviour so that it displays notifications for all messages it gets when the stream reconnects, eg. after you wake a laptop up from sleep. Is this desired?